src/components/DocumentationLayout.vue

```diff
@@ -150,12 +152,21 @@ export default {
       sidenavHiddenOnLarge: storage.get(NAVIGATOR_HIDDEN_ON_LARGE_KEY, false),
       showQuickNavigationModal: false,
       BreakpointName,
+      indexState: IndexStore.state,
     };
   },
   computed: {
     enableQuickNavigation: ({ isTargetIDE }) => (
       !isTargetIDE && getSetting(['features', 'docs', 'quickNavigation', 'enable'], true)
     ),
+    quickNavData({ indexState, interfaceLanguage }) {
+      let currentLangIndexData = indexState.flatChildren[interfaceLanguage] || [];
```

Comment:
It seems like this is basically just looking up the nodes for the current language and falling back to the Swift ones as a default/backup, so maybe this could be cleaned up a bit? I'm thinking something like:

```javascript
indexNodes: ({ indexState: { flatChildren }, interfaceLanguage }) => (
  flatChildren[interfaceLanguage] ?? flatChildren[Language.swift.key.url]
),
```

What do you think?